src/client/features/interactions/index.js

```diff
@@ -51,12 +52,18 @@ class Interactions extends React.Component {
         Phosphorylation:true,
         Expression:true
       }
-    };
+    };    
 
     const query = queryString.parse(props.location.search);
-    const sources=_.uniq(_.concat([],query.source));
-    const kind = sources.length>1?'PATHSBETWEEN':'Neighborhood';
-    ServerAPI.getNeighborhood(sources,kind).then(res=>{
+    const sources = _.uniq(_.concat([],query.source)); //IDs or URIs
+    const params = {
+      source : sources,
+      pattern : ['controls-phosphorylation-of','in-complex-with','controls-expression-of', 'interacts-with'],
+      kind : sources.length>1 ? 'pathsbetween' : 'neighborhood',
```

Comment:
Is it the case that 'pathsbetween' will not return 'interacts-with'?

Reply:
It says "get ids from uris" in the comment line above that code (if, e.g., an uri is standard PR's uri, I bet); @logmosier can comment on exact purpose of this. I did not change the purpose of this code. It would be great to have more comments in the code, in many places.

My changes (using one universal new method `ServerAPI.pcQuery` instead of `ServerAPI.getNeighborhood`): https://github.com/PathwayCommons/app-ui/commit/bc43b132985cc406cb9502ac9165ee73cd8a0d69#diff-48d6e306661f0412cd8851fd566e8bba
`ServerAPI.getNeighborhood` (removed by https://github.com/PathwayCommons/app-ui/commit/bc43b132985cc406cb9502ac9165ee73cd8a0d69#diff-0e447967aaeb299d989022e921439ad2) used the same graph query parameters.

PS (off-topic):
Also, look how I filter datasources for that drop-down menu (to keep only those actually having pathways): https://github.com/PathwayCommons/app-ui/commit/9b745adb006efe2f6ae8e81a2d82f10334e8250f#diff-0e447967aaeb299d989022e921439ad2